api4/plugin_test.go

```diff
@@ -58,8 +58,11 @@ func TestPlugin(t *testing.T) {
 	assert.Equal(t, "testplugin", manifest.Id)
 
 	// Stored in File Store: Install Plugin from URL case
-	pluginStored, _ := th.App.FileExists("./plugins/" + manifest.Id + ".tar.gz")
-	assert.True(t, pluginStored)
+	if pluginStored, err := th.App.FileExists("./plugins/" + manifest.Id + ".tar.gz"); err != nil {
```

Comment:
same comment as below, better to do something like

```
pluginStored, err := th.App.FileExists("./plugins/" + manifest.Id + ".tar.gz");
assert.Nil(t, err)
assert.True(t, pluginStored)
```



https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow

Reply:
Very useful reference!  Thanks for the link 👍